tasks/main.yml

```diff
@@ -4,19 +4,10 @@
 # Set up defaults to set bools to false
 
 - block:
-  - name: "Remove file"
```

Comment:
This originally came from the collectd ansible, but I'm not sure its purpose. If I remove it, my playbook passes impotence tests, which is important IMO.

I could not figure out a good reason to remove this file when the template directive is going to update it anyways.

Reply:
In collectd, there is a config generated per plugin, and if a plugin is removed from the plugins list it is not removed from the conf dir if collectd-config is run again, so we're left with extra plugins that have not been explicitly enabled.

Reply:
Ah, right! You may have explained this to me on another PR as well, sorry. :) In the QDR case, I think this will be just fine. Thanks!